### PR TITLE
build: create exchange-rates worker and durable object

### PR DESCRIPTION
**Pull request by mgnsharon**

### CODE REVIEW

File: packages/monitors/src/exchange-rates-mainnet.ts [hunk 1]
```diff
@@ -120,27 +118,43 @@ async function aggregateCalls() {
...
+
     provider = getProvider('mainnet');
     const { network, blockNumber, results } = await aggregateCalls();
-
+    const req = new Request(`${env.EXCHANGE_RATE_URL}/exchange-rates`, {
+      method: 'PUT',
+      body: JSON.stringify({ rates: { network, blockNumber, results } }),
+    });
+    const resp = await stub.fetch(req);
+    console.log(JSON.stringify(resp));
```
Comment (jeffywu) [inline packages/monitors/src/exchange-rates-mainnet.ts:135]: Remove this console.log?

File: apps/blockchain-monitor/src/index.ts [hunk 2]
```diff
@@ -18,13 +23,17 @@ export default {
...
+      try {
+        ctx.waitUntil(
+          await Promise.all(
+            getJobs(env.NX_ENV)
+              .get(controller.cron)
+              .map((m) => m.run({ ctx, env }))
+          )
+        );
+      } catch (error) {
+        console.log((error as Error).message);
```
Comment (jeffywu) [inline apps/blockchain-monitor/src/index.ts:35]: Should this log to datadog?

Comment (mgnsharon) [reply apps/blockchain-monitor/src/index.ts:35]: yeah probably, I added the try/catch to debug, I should prob make it official

File: packages/monitors/src/exchange-rates-mainnet.ts (hunk 1 above)
Comment (mgnsharon) [reply packages/monitors/src/exchange-rates-mainnet.ts]: done

**Pull request merged**